README.rst

```diff
@@ -85,20 +85,65 @@ How to use this tool
 1. git clone this repo:
 
    ::
-   
+
       git clone https://github.com/ios-xr/iosxrv-x64-vbox.git
 
-2. Install VirtualBox and Vagrant (see guide below).
+2. Install VirtualBox, Vagrant and socat (see guide below).
 3. Download the appropriate ISO file, e.g. ``iosxrv-fullk9-x64.iso``
```

Comment:
@glennmatthews Ran this overnight - 10x manually and several times in Jenkins and successfully build and passed ut each time.

https://engci-jenkins-rtp.cisco.com/jenkins/job/team_appdevci/job/iosxrv64_iso2vbox/37/console